packages/office-ui-fabric-react/src/components/Nav/Nav.base.tsx

```diff
@@ -221,7 +221,12 @@ export class NavBase extends BaseComponent<INavProps, INavState> implements INav
     });
 
     return (
-      <button className={classNames.chevronButton} onClick={this._onGroupHeaderClicked.bind(this, group)}>
+      <button
+        className={classNames.chevronButton}
+        onClick={this._onGroupHeaderClicked.bind(this, group)}
+        aria-label={this.props.expandButtonAriaLabel}
```

Comment:
Nit: consider adding `expandButtonAriaLabel` to the destructured `this.props` above